include/ONV.hpp

```diff
@@ -0,0 +1,144 @@
+#ifndef GQCG_ONV_HPP
+#define GQCG_ONV_HPP
+
+
+#include <Eigen/Dense>
+#include "common.hpp"
+
+
+
+namespace GQCG {
+
+
+class ONV {
+private:
+    const size_t K;  // number of spatial orbitals
+    const size_t N;  // number of electrons
+    size_t unsigned_representation;  // unsigned representation
+    VectorXs occupation_indexes;  // the occupied orbital electron indexes
```

Comment:
add documentation (maybe graphical, inside the ONV class documentation) about occupation_indices

rename to `occupation_indices`